packages/Akvo_rabbitmq_client/tests/test_queue_message_util.py

```diff
@@ -6,12 +6,15 @@
 class TestQueueMessageUtil(unittest.TestCase):
 
     def test_create_queue_message_with_minimal_data(self):
+        headers = {"reply_to": "twilio"}
```

Comment:
With this merge the reply-to pattern is no longer needed. I have already removed it from the design document. Rather, we have the `platform` field that can be used in the backend to route the message back to the right phone number. So when the platform is `WHATSAPP`, send the message to the recipient via Twilio.

Reply:
So this mean, we will remove the twiliobot and slackbot replies queue right?

Reply:
Correct. You only need a queue that leads to the backend.

Maybe that change is best done on the next, Slackbot removal branch. So leave it for now and take it into the next PR.